### ISSUE TITLE
Checkpoint crashing upon opening

### ISSUE THREAD

**Issue by adamarsx** [FlagBrew/Checkpoint#186]
**Describe the bug**
Immediately after launching, the Switch crashes and tells me to reboot.
**To Reproduce**
Steps to reproduce the behavior: Just launch Checkpoint from hbmenu

**Switch:**
- System version: 6.2.0 (latest Atmosphere)
- Checkpoint version: 3.6.0
- SD Card Filesystem: FAT32
- Result code (if available): 2345-0031

**Additional context**
This might be macOS related (I've been having some problems with hbmenu recognizing some folders and am currently running all nro files straight from the root of my SD)

**Comment by Xiaodx912** [comment]
The same problem.

- system:6.20

- SX OS:2.5.2 with emuNAND enable

- filesystem : exfat

The first time I use macOS and got several problems (hbmenu can't find /switch etc.) 
Then I turn to windows ,delete all the hide file created by mac , copy all files out of tf card , formated the card then copy them back , but problems remains.

**Comment by adamarsx** [comment]
I am using FAT32

**Comment by BernardoGiordano** [comment]
Which hbmenu version are you using?

**Comment by adamarsx** [comment]
I've tried 3.0.0 and 3.0.1, both can't access the /switch folder (all other folders are usable), I've tried running commands from terminal, a Windows VM and Hekate to disable the archive bit, but the folder still can't be recognised. I've also tried deleting the folder and creating a new one from the Windows VM which also doesn't work. NX-Shell detects a 0B file called SWITCH on the root of my SD, which it can't delete.
